tests/test_infoservice.py

```diff
@@ -18,7 +20,7 @@ class TestInfoClient:
 
     @responses.activate
     def test_infoclient(self, myclient, mocker):
-        endpoint_mapping = self.default_mapping
+        endpoint_mapping = self.default_mapping  # noqa F841
```

Comment:
@ceesem this is giving `F841 Local variable `endpoint_mapping` is assigned to but never used` but i opted to leave it because of what we just discussed w pytest. lmk if you think this is really not used, though